docs/releases.md

```diff
@@ -1,6 +1,39 @@
 # Ixia-c Release Notes and Version Compatibility
 
-## Release v0.0.1-1622 (Latest)
+## Release v0.0.1-2120 (Latest)
+> 27th August, 2021
+
+#### About
+
+This build contains support for capture filter, setting GRE checksum flag, receiving controller log in stdout and some bug fixes.
+
+#### Build Details
+
+| Component                     | Version       |
+|-------------------------------|---------------|
+| Open Traffic Generator API    | [0.4.12](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.4.12/openapi.yaml)         |
+| snappi                        | 0.4.25        |
+| ixia-c-controller             | 0.0.1-2120    |
+| ixia-c-traffic-engine         | 1.4.0.9       |
+| ixia-c-app-usage-reporter     | 0.0.1-37      |
+
+#### New Feature(s)
+
+* Capture filter is now supported. One or muiltiple patterns can be added when setting the config.
```

Comment:
Capture filters are now supported.  Multiple patterns can be specified in the configuration.

Reply:
Modified.